chord_metadata_service/restapi/api_renderers.py

```diff
@@ -117,18 +117,18 @@ def generate_csv_response(data, filename, columns):
     return response
 
 
-def render_individual_age(individual):
-    if "time_at_last_encounter" not in individual:
+def render_age(individual, time_key):
+    if time_key not in individual:
         return None
-    time_at_last_encounter = individual["time_at_last_encounter"]
+    time_encounter = individual[time_key]
```

Comment:
this would be better called `time_to_render` or something... since it could be collection time and not encounter time